index.js

```diff
@@ -124,10 +125,78 @@ function parseFileExtension(opts) {
   return opts;
 }
 
+// mongodb-linux-x86_64-rhel62-3.6.2.tgz
+//
+// Red Hat Enterprise Linux Server release 7.2 (Maipo)
+function parseRHEL(data) {
+  if (data.toString().match('7.1')) return 'rhel71';
```

Comment:
please call `toString` once at the top, otherwise you're converting the buffer to a string with every check here

Reply:
Realized I'm actually calling `toString` already when passing the argument in so I don't need it here at all 👍